app/views/posts/show.html.erb

```diff
@@ -1,2 +1,37 @@
-<h1>Posts</h1>
-<p>Here is a post that was created</p>
+<div class='post-comment-wrapper container'>
+    <div class='post-wrapper'>
+        <div class='post-div'>
+            <h4>Post #1 </h4>
```

Comment:
- [ ] It would be better if we can retrieve the 5 most recent comments `post.recent_comments`. It is better to interact with the database instead of hard coding most / all the data